README.md

```diff
@@ -1,9 +1,11 @@
 # Awesome Visual Regression Testing [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+
```

Comment:
Style-Changes should not be mixed up with content-contributions. If you think this new lines are neccessary (which I don't) please move them to an seperate pull-request.